File: searching/Linear_Search.py
def LinearSearch(a, size, key):
    for i in range(size):
        if a[i] == key :
            return i
    return -1

if __name__ == "__main__":
    a = []
    n = int(input("Enter the number of elements in array :"))
    print("Enter the elements in array :")
    for i in range(n):
        a.append(int(input()))
    print("Array is :", a)
    key = int(input("Enter the element to be searched :"))
    result = LinearSearch(a, n, key)
    if result == -1 :
        print("Element not found")
    else:
        print("Element found at index : {}".format(result))

File: searching/linearSearch.java
public class LinearSearch{    
public static int linearSearch(int[] arr, int key){    
        for(int i=0;i<arr.length;i++){    
            if(arr[i] == key){    
                return i;    
            }    
        }    
        return -1;    
    }    
    public static void main(String args[]){    
        int[] a= {10,20,30,50,70,90};    
        int key = 50;    
        System.out.println(key+" is found at an index: "+linearSearch(a, key));    
    }    
}   